system/HTTP/URI.php

```diff
@@ -787,51 +776,59 @@ protected function filterPath(string $path=null)
 		$path = $this->removeDotSegments($path);
 
 		// Fix up some leading slash edge cases...
-		if (strpos($orig, './') === 0) $path = '/'. $path;
-		if (strpos($orig, '../') === 0) $path = '/'. $path;
+		if (strpos($orig, './') === 0)
+			$path	 = '/'.$path;
+		if (strpos($orig, '../') === 0)
+			$path	 = '/'.$path;
 
 		// Encode characters
 		$path = preg_replace_callback(
-			'/(?:[^' . self::CHAR_UNRESERVED . ':@&=\+\$,\/;%]+|%(?![A-Fa-f0-9]{2}))/',
-			function (array $matches)
-			{
-				return rawurlencode($matches[0]);
-			},
-			$path
+			'/(?:[^'.self::CHAR_UNRESERVED.':@&=\+\$,\/;%]+|%(?![A-Fa-f0-9]{2}))/', function (array $matches)
+		{
+			return rawurlencode($matches[0]);
+		}, $path
 		);
 
 		return $path;
 	}
 
 	//--------------------------------------------------------------------
 
-
-
 	/**
 	 * Saves our parts from a parse_url call.
 	 *
 	 * @param $parts
 	 */
 	protected function applyParts($parts)
 	{
-		if (! empty($parts['host'])) $this->host = $parts['host'];
-		if (! empty($parts['user'])) $this->user = $parts['user'];
-		if (! empty($parts['path'])) $this->path = $this->filterPath($parts['path']);
-		if (! empty($parts['query'])) $this->query = $this->filterQuery($parts['query']);
-		if (! empty($parts['fragment'])) $this->fragment = $this->filterQuery($parts['fragment']);
+		if (!empty($parts['host']))
```

Comment:
This is just what the NB formatting did. I don't recall changing any setting for this.
I could be mistaken, as there seven sets of formatting option pages.

It looks like there are five or so rules for wrapping with control structures, and that wasn't addressed in our style guide, IIRC. They each can be set to "never", "always", or "if long". The default is "always", and should probably be "if long".